packages/server/src/bandit/banditSelection.ts

```diff
@@ -0,0 +1,75 @@
+import { EpicTest, EpicVariant } from '@sdc/shared/dist/types';
+import { BanditData } from './banditData';
+import { Result } from '../tests/epics/epicSelection';
+import { putMetric } from '../utils/cloudwatch';
+import { logError } from '../utils/logging';
+
+/**
+ * Select at random with probability epsilon.
+ * https://en.wikipedia.org/wiki/Multi-armed_bandit#Semi-uniform_strategies
+ */
+const EPSILON = 0.1;
+
+export function selectVariantWithHighestMean(
+    testBanditData: BanditData,
+    test: EpicTest,
+): EpicVariant | undefined {
+    const variant =
+        testBanditData.bestVariants.length < 2
+            ? testBanditData.bestVariants[0]
+            : testBanditData.bestVariants[
+                  Math.floor(Math.random() * testBanditData.bestVariants.length)
+              ];
+
+    if (!variant) {
+        return undefined;
+    }
+
+    return test.variants.find((v) => v.name === variant.variantName);
```

Comment:
Presumably it won't be possible to have variants with the same name?

Reply:
the RRCP prevents this so 🤞  it shouldn't be possible!

Reply:
All good then!